src/writing-your-own-application/alloc.md

```diff
@@ -0,0 +1,53 @@
+# Alloc
+
+In `no_std`, we need to rely on `alloc` crate, which provides heap allocation support. To use `alloc` in `no_std` environment:
+
+1. Explicitly enable the alloc crate
```

Comment:
I'm not sure what "enabling" the crate is meant to mean in this context; further detail would be nice.

Reply:
I removed this as I added a step-by-step what needs to be done